packages/react/src/components/Forms/Select/multi-common.tsx

```diff
@@ -168,6 +168,7 @@ export const SelectMultiAux = ({
                         aria-label={t(
                           "components.forms.select.clear_button_aria_label",
                         )}
+                        type="button"
```

Comment:
We could add this as a default value in `Button`.
Either of both solution are ok for me.